e2e/terraform/provision-infra/userdata/windows-2022.ps1

```diff
@@ -0,0 +1,180 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: BUSL-1.1
+
+<powershell>
+
+Set-StrictMode -Version latest
+$ErrorActionPreference = "Stop"
+
+$RunningAsAdmin = ([Security.Principal.WindowsPrincipal] [Security.Principal.WindowsIdentity]::GetCurrent()).IsInRole([Security.Principal.WindowsBuiltInRole] "Administrator")
+if (!$RunningAsAdmin) {
+  Write-Error "Must be executed in Administrator level shell."
+  exit 1
+}
+
+# -------------------------------------------
+# Disks and Directories
+
+# Bring ebs volume online with read-write access
+Get-Disk | Where-Object IsOffline -Eq $True | Set-Disk -IsOffline $False
+Get-Disk | Where-Object isReadOnly -Eq $True | Set-Disk -IsReadOnly $False
+
+New-Item -ItemType Directory -Force -Path C:\opt -ErrorAction Stop
+New-Item -ItemType Directory -Force -Path C:\opt\nomad
+New-Item -ItemType Directory -Force -Path C:\etc\nomad.d
```

Comment:
line 22 creates `C:\opt`, but this presumes that `C:\etc` already exists?

Reply:
Done!